.github/workflows/pa11y.yml

```diff
@@ -21,13 +21,10 @@ jobs:
           node-version: 16
 
       - name: Install dependencies
-        run: npm ci
+        run: npm ci && npm install gulp && npm install -g gulp
```

Comment:
`run: npm run generate`
seemed to work for me

Reply:
yup.  Worked perfectly.  THANK YOU!!!